### ISSUE TITLE
Collection extent: End inclusive or exclusive

### ISSUE THREAD

**Issue by m-mohr** [opengeospatial/ogcapi-features#934]
Is the end timestamp that is specified in the extents.temporal.interval property of a Collection as second element of each subarray inclusive or exclusive? I can't find it defined in the specification.

**Comment by cportele** [comment]
You are correct, it is not clearly stated. For me it was always clear that for both the spatial and temporal extent that the ends are inclusive. 

The current requirement is worded as "each feature in the collection SHALL be inside the extent described by the first time interval in the `interval` array." The wording is similar for the spatial extent. 

I think, instead of "inside" we should use the spatial and temporal predicates from CQL2 to state the requirement, in particular since CQL2 is now approved. I will create a PR.

As an aside, in CQL2 intervals are closed at both ends.

**Comment by cportele** [comment]
Meeting 2024-07-01: @cportele will create a pull request as proposed in his comment.

**Comment by m-mohr** [comment]
Thank you, Clemens. That's a very welcome clarification as we were discussing that in STAC recently.
